src/data/nat/choose/basic.lean

```diff
@@ -267,4 +267,21 @@ begin
     apply zero_le }
 end
 
+-- Inequalities about increasing the first argument
+
+lemma choose_le_add_one (a c : ℕ) : choose a c ≤ choose (a + 1) c :=
+by cases c; simp [nat.choose_succ_succ]
```

Comment:
```suggestion
lemma choose_le_succ (a c : ℕ) : choose a c ≤ choose a.succ c :=
by cases c; simp [nat.choose_succ_succ]
```
since `choose_le_add a 1 c` already has statement `choose a c ≤ choose (a + 1) c`; this one at least might be useful for someone with a `succ` in their goal.

Reply:
done!